src/pages/Admin/Venue/Details/VenueDetails.tsx

```diff
@@ -153,15 +153,10 @@ const VenueDetails: React.FC<VenueDetailsProps> = ({ venue }) => {
   if (!user) return null;
 
   const handleEditRoomSave = async (values: RoomData_v2, index: number) => {
-    const newData = {
+    const newData: RoomData_v2 = {
+      ...editingRoom,
       ...values,
-      x_percent: editingRoom?.x_percent,
-      y_percent: editingRoom?.y_percent,
-      width_percent: editingRoom?.width_percent,
-      height_percent: editingRoom?.height_percent,
-      isEnabled: editingRoom?.isEnabled,
```

Comment:
Pending satisfactory resolution of investigating https://github.com/sparkletown/sparkle/pull/1300#discussion_r609102083